src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -20,55 +21,73 @@ public class CodeStyleOptions
         /// This option says if we should simplify away the <see langword="this"/>. or <see langword="Me"/>. in field access expressions.
         /// </summary>
         public static readonly PerLanguageOption<CodeStyleOption<bool>> QualifyFieldAccess = new PerLanguageOption<CodeStyleOption<bool>>(nameof(CodeStyleOptions), nameof(QualifyFieldAccess), defaultValue: CodeStyleOption<bool>.Default,
-            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.QualifyFieldAccess"));
+            storageLocations: new OptionStorageLocation[]{
+                new EditorConfigStorageLocation("dotnet_style_qualification_for_field", ParseEditorConfigCodeStyleOption),
```

Comment:
I'm not morally against teaching EditorConfigStorageLocation about CodeStyle options directly to avoid having to pass the parser to each one specifically. We did that in the RoamingProfileStorageLocation as well.